cpp/add_title_keywords.cc

```diff
@@ -182,30 +173,31 @@ int main(int argc, char **argv) {
     const std::string marc_input_filename(argv[verbose ? 2 : 1]);
     const std::string marc_output_filename(argv[verbose ? 3 : 2]);
     if (unlikely(marc_input_filename == marc_output_filename))
-        logger->error("MARC input file name equals MARC output file name!");
+        LOG_ERROR("MARC input file name equals MARC output file name!");
 
-    std::unique_ptr<MarcReader> marc_reader(MarcReader::Factory(marc_input_filename, MarcReader::BINARY));
-    std::unique_ptr<MarcWriter> marc_writer(MarcWriter::Factory(marc_output_filename, MarcWriter::BINARY));
+    auto marc_reader(MARC::Reader::Factory(marc_input_filename));
```

Comment:
Do we know why BINARY was hardcoded here?  Maybe it's part of a pipeline reading from a FIFO?

Reply:
Apparently, this tool is never used.